src/Schema/Comparator.php

```diff
@@ -21,23 +21,29 @@
 class Comparator
 {
     /**
+     * Returns a SchemaDiff object containing the differences between the schemas $fromSchema and $toSchema.
+     *
+     * The returned differences are returned in such a way that they contain the
+     * operations to change the schema stored in $fromSchema to the schema that is
+     * stored in $toSchema.
```

Comment:
It looks like this paragraph belongs more to the `SchemaDiff` class itself rather than this method. Would it make sense to move it here?